pkg/codegen/templates/chi/chi-middleware.tmpl

```diff
@@ -177,9 +177,15 @@ func (siw *ServerInterfaceWrapper) {{$opid}}(w http.ResponseWriter, r *http.Requ
     siw.Handler.{{.OperationId}}(w, r{{genParamNames .PathParams}}{{if .RequiresParamObject}}, params{{end}})
   })
 
+  {{if opts.Compatibility.ApplyChiMiddlewareFirstToLast}}
```

Comment:
I must be missing something - how does `opts` get passed in here? When we call GenerateChiServer, we pass in an array of OperationDefinition, which doesn't have opts.

Reply:
Oh, nevermind, I see it. This code has changed so much since I wrote it, I get lost sometimes.